### PR TITLE
Portfolio Contact Form Check-up

### PR DESCRIPTION
**Pull request by frankfembo**
- Add a contact form to portfolio project with input fields
- Include submit button to send user input details to formspree
- Add a footer & mailto: at bottom

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -244,7 +244,7 @@ <h4 class="con-us">CONTACT US</h4>
                 </li>
                 <li class="li-item">
                   <label for="textarea">
-                    <textarea name="text" id="text"  class="text-input detail-input" placeholder="Enter text here..." cols="23" rows="5" max-length="500" required></textarea>
+                    <textarea class="text-input detail-input" name="text" id="text" cols="23" rows="5" placeholder="Enter text here..." max-length="500" required></textarea>
```
Comment (topeogunleye) [inline index.html:247]: - [ ] I kindly suggest that you correct your spelling of maxlength HTML attribute

:x: Wrong
`max-length="500"`

:heavy_check_mark: Right
`maxlength="500"`

File: index.html [hunk 2]
```diff
@@ -244,7 +244,7 @@ <h4 class="con-us">CONTACT US</h4>
...
                   </label>
                 </li>
                 <li class="btn-container">
```
Comment (topeogunleye) [inline index.html:250]: - [ ] I kindly suggest that you add the different button interactions including active (when it's pressed).

image

the button on the pressed state has the `background: #5E42D7;`
### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #5E42D7;
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @frankfembo,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:

>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @frankfembo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**